libnetwork/iptables/iptables_test.go

```diff
@@ -46,59 +44,6 @@ func TestNewChain(t *testing.T) {
 	createNewChain(t)
 }
 
-func TestForward(t *testing.T) {
```

Comment:
Could you preserve this test while moving code to the `bridge` pkg? Or maybe don't move that code in the first place (see my other comment about that).

Reply:
Splitting apart the DNAT/masquerade and filter rules is a main objective of this change, they're now controlled separately... so, the `Forward` function tested here no longer exists. There's better coverage in `TestAllocatePorts`.